packages/api/src/controllers/stream.ts

```diff
@@ -770,6 +781,7 @@ app.post(
       profiles: childStream.profiles,
       record,
       recordObjectStoreId,
+      recordingSessionId,
```

Comment:
Yeah, IMO this should be just "session ID" and it should be used as the `id` field above. We should not use the same "child stream" ID as the ID of the session amymore (especially since they're 1-N now, not 1-1). WDYT?

Reply:
I think we still need to keep the relation 1-1 between child stream and sessions because we make this assumption in a lot of code parts by passing child stream as a session, e.g. [here](https://github.com/livepeer/studio/blob/cbdbc0d335d132c5b6610c17d60e4fac687e9a1b/packages/api/src/controllers/stream.ts#L453) and [here](https://github.com/livepeer/studio/blob/cbdbc0d335d132c5b6610c17d60e4fac687e9a1b/packages/api/src/controllers/stream.ts#L593).

In general, I think we could change this relation to 1-N and reuse sessions (while not reusing child streams), but I think we would need to make some bigger refactoring/change. So, I'd leave it as a separate PR. 

Wdyt? Or am I missing something?

Reply:
This is the main requirement for this change in Studio API! To have a consistent session object that corresponds to the "Mist session" (and thus, recording). If we don't make this change, there will be several other places that will break, like for example now creating N assets from 1 recording cause we have potentially multiple sessions per "mist session". The 1-1 relationship hasn't existed [for more than a month](https://github.com/livepeer/studio/pull/1662) and is only like this now to make it simpler to create the right 1-N relationship that we want (it was 1-N before but in a broken way).

So IMO that refactoring is exactly what we should be doing in this PR! Otherwise we're gonna have to solve a lot of other problems to go in a direction that we don't really wanna go.